apps/wallet/src/background/keyring/VaultStorage.noSessionStorage.test.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { describe, expect, it, vi } from 'vitest';
+import Browser from 'webextension-polyfill';
+
+import { VaultStorage } from './VaultStorage';
+import { SESSION_STORAGE } from './storage-utils';
+import { testVault } from '_src/test-utils/vault';
+
+vi.mock('./storage-utils', () => ({ SESSION_STORAGE: null }));
+
+describe('VaultStorage no session storage', () => {
```

Comment:
Can you not unify the no session storage and the session storage test files by just restoring mocks in some `after`?

Reply:
I will refactor VaultStorage and that would help to unify this 👍